Sources/KeyboardShortcuts/Localization/String++.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  String++.swift
+//  
+//
+//  Created by Licardo on 2020/8/28.
+//
+
+import Foundation
+
+extension String {
+    var localized: String {
+        NSLocalizedString(self, bundle: .module, comment: self)
+    }
+}
```

Comment:
I prefer to have this in the `util.swift` file.

Reply:
Done